### ISSUE TITLE
Filters working together

### ISSUE THREAD

**Issue by JadedRain** [JadedRain/Incrementum#391]
Update how stocks are screened to demonstrate that multiple filters can be selected at once. The expected behavior is to first limit the stocks down to a select few ex [INTC, TSLA, NVDA, F] and apply the filter for stocks in the motorvehicle industry. With both of these applied the list of stocks shown as results should be [TSLA, F]

Acceptance Criteria:
Screener class logic is updated to hand filtering by individual symbols and industry metadata
Demo screener page is updated and should be able to apply both filters such that the expected behavior above occurs
Create a test case that captures and preserves this behavior using the scenario listed in the description
